### PR TITLE
Create PrateekSrivastava.md

### PR DESCRIPTION
**Pull request by Prateek-Srivastav**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
Please add your name to Data.json

**Comment by Prateek-Srivastav** [comment]
done!

File: journeys/PrateekSrivastava.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+📌 I am co founder of [AceDrops](https://acedrops.in) (handcrafted items with the customizations you want with the comfort of home😊) and Qualido (we sell books without cutting much from your pocket😉).
+
+📌 I worked as a core team member in startups too.
+
+
+
+# Projects i have worked on
+
+- created a fully functioning job searching and placement android application for a startup.
```
Comment (KendallDoesCoding) [inline journeys/PrateekSrivastava.md:15]:
```suggestion
- Created a fully functioning job searching and placement android application for a startup.
```

File: journeys/PrateekSrivastava.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- created backend of an e-commerce like startup from scratch involving all kindof functioning like CRUD requests, searching and hosting on AWS.
```
Comment (KendallDoesCoding) [inline journeys/PrateekSrivastava.md:16]:
```suggestion
- Created backend of an e-commerce like startup from scratch involving all kind of functioning like CRUD requests, searching and hosting on AWS.
```

File: journeys/PrateekSrivastava.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- created other android applications as a project like recipe app and the shop app.
```
Comment (KendallDoesCoding) [inline journeys/PrateekSrivastava.md:17]:
```suggestion
- Created other android applications as a project like recipe app and the shop app.
```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: journeys/PrateekSrivastava.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+
+# Other than coding , i love
```
Comment (KendallDoesCoding) [inline journeys/PrateekSrivastava.md:20]:
```suggestion
# Other than coding , I love
```

File: journeys/PrateekSrivastava.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+Other than that ,
+
+📌 I am co founder of [AceDrops](https://acedrops.in) (handcrafted items with the customizations you want with the comfort of home😊) and Qualido (we sell books without cutting much from your pocket😉).
+
+📌 I worked as a core team member in startups too.
+
+
+
+# Projects i have worked on
```
Comment (KendallDoesCoding) [inline journeys/PrateekSrivastava.md:13]:
```suggestion
# Projects I have worked on
```

**Review by KendallDoesCoding — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
Thanks for your contribution @Prateek-Srivastav! Woo-Hoo Your jourrney is now added

**Pull request merged**